src/app/features/auth/types/authRepository.ts

```
@@ -0,0 +1,7 @@
+export interface AuthRepository {
+  saveToken(token: string, email: string): Promise<void>;
+  getEmailByToken(token: string): Promise<string | null>;
+  hasToken(token: string): Promise<boolean>;
+  connect(): Promise<void>;
+}
+export interface AuthApi {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)